runtime/integration-tests/src/xcm/kusama/restricted_calls.rs

```diff
@@ -0,0 +1,114 @@
+// This file is part of Altair chain project.
+//
+// Altair is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version (see http://www.gnu.org/licenses).
+// Altair is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// Copyright 2021 Altair GmbH (altair.io).
+// This file is part of Altair chain project.
+//
+// Altair is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version (see http://www.gnu.org/licenses).
+// Altair is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+use crate::xcm::kusama::setup::{
+	air, foreign, sibling_account, CurrencyId, ALICE, BOB, PARA_ID_SIBLING,
+};
+use crate::xcm::kusama::test_net::{Altair, KusamaNet, Sibling, TestNet};
+use altair_runtime::{Balances, Call, CustomMetadata, Origin, PolkadotXcm, XTokens};
+use frame_support::dispatch::Dispatchable;
+use frame_support::{assert_err, assert_noop, assert_ok};
+use orml_traits::{asset_registry::AssetMetadata, FixedConversionRateProvider, MultiCurrency};
+use runtime_common::xcm_fees::{default_per_second, ksm_per_second};
+use runtime_common::{decimals, parachains, Balance, XcmMetadata};
+use sp_runtime::DispatchError;
+use sp_runtime::DispatchError::BadOrigin;
+use xcm::latest::{Junction, Junction::*, Junctions::*, MultiLocation, NetworkId};
+use xcm::v1::MultiAsset;
+use xcm::v2::{AssetId, Fungibility};
+use xcm::v2::{Instruction::WithdrawAsset, Xcm};
+use xcm::VersionedMultiLocation;
+use xcm_emulator::TestExt;
+
+/// Verify that calls that must be blocked by the BaseCallFilter are indeed blocked.
+pub mod blocked {
+	use super::*;
+
+	#[test]
+	fn xtokens_transfer() {
+		// For now, Tranche tokens are not supported in the XCM config so
+		// we just safe-guard that trying to transfer a tranche token fails.
+		// Once Tranche tokens are supported, we need to tighten this test.
+		Altair::execute_with(|| {
+			assert!(XTokens::transfer(
```

Comment:
should this be an `assert_noop(...)` instead of `assert(...is_err())`?

Reply:
The reason why I went with `assert(...is_err())` here instead of checking for a specific error is that in this case we don't care what the error is as long as it fails. But if you would prefer to go with `assert_noop` we can do that! let me know